lib/web/ui/app.go

```diff
@@ -36,7 +36,7 @@ type App struct {
 	Kind string `json:"kind"`
 	// SubKind is the subkind of the app resource. Used to differentiate different
 	// flavors of app.
-	SubKind string `json:"subkind,omitempty"`
+	SubKind string `json:"subKind,omitempty"`
```

Comment:
it looks like we havent used this on the web client (it seems we just added it in this PR ) but we may not want to change this casing in case some other clients depend on it. im not 100% sure if any other client is using it or how our backward compatibilty agreement works in that regard, but i'd double check this first.

Reply:
Thanks for double checking but we are safe here :)
This new `subKind` field was added in the depending backend PR https://github.com/gravitational/teleport/pull/49301/files#diff-f635fca1ca174ba271760ef6e8dbe4aa703df1ba9acde1ae8828b86fb8992631R39 for the sole purpose of supporting changes in this PR.